src/System.Collections.NonGeneric/tests/System.Collections.NonGeneric.Tests.csproj

```diff
@@ -8,6 +8,8 @@
     <RootNamespace>System.Collections.NonGeneric.Tests</RootNamespace>
     <AssemblyName>System.Collections.NonGeneric.Tests</AssemblyName>
     <ProjectGuid>{EE95AE39-845A-42D3-86D0-8065DBE56612}</ProjectGuid>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
+    <TargetFrameworkProfile />
```

Comment:
Can this be conditioned so that if the requirements aren't met (i.e. you aren't building with VS 2015 + Universal Windows App Dev Tools) the perf tests are omitted from the build?  Make it pay-for-play in the short term, and document how to set up a build environment to enable building perf tests.  We should also ensure that this will work with VS 2015 Community Edition.

CC @joshfree